templates/app/kafka-secrets.yaml

```diff
@@ -15,11 +15,15 @@ objects:
     username: "${MESSAGING_USERNAME}"
 parameters:
 - name: APP_NAME
+  description: The name of the application that ManageIQ will be deployed as in OpenShift.  Used to group OpenShift objects.
```

Comment:
Can you remove the `ManageIQ` name from these comments as we try to keep that configurable.
(Hence APP_NAME :trollface: )